test/data/crypto/CoreBuffer.ts

```diff
@@ -41,11 +41,7 @@ export class CoreBuffer extends Serializable implements ICoreBuffer {
         return Buffer.from(this._buffer).toString("base64")
     }
 
-    public static deserialize(value: string): CoreBuffer {
-        return CoreBuffer.from(value)
-    }
-
     public static from(value: any): CoreBuffer {
-        return new CoreBuffer(value)
```

Comment:
Ich bin mir auf den ersten Blick nicht sicher, ob das auch tatsächlich so funktionieren kann. Wir geben über from/fromAny doch niemals in den Constructor etwas mit. Insofern wird der constructor von CoreBuffer nie aufgerufen und das was als Wert reinkommt nie zu einem UInt8Array konvertiert?